Letterbook.Adapter.Db/EntityConfigs/ConfigureFollowerRelations.cs

```diff
@@ -8,10 +8,8 @@ public class ConfigureFollowerRelations : IEntityTypeConfiguration<Models.Follow
     public void Configure(EntityTypeBuilder<Models.FollowerRelation> builder)
     {
         builder.HasKey(relation => relation.Id);
+        builder.Property(relation => relation.Id)
+            .ValueGeneratedNever();
```

Comment:
This was a nightmare to figure out. This entity wasn't writing to the DB, but the only error was something like `The database operation was expected to affect 1 row(s), but actually affected 0 row(s)`

I've been researching and debugging it for 2 days. I eventually found this SO post with the [explanation and solution](https://stackoverflow.com/a/74932577/4100159). The problem was that we generate our own PKeys for FollowerRelations, because it's a GUID so we can. But, EFCore interpreted that as being an existing entity that we need to update, rather than a new one to insert. So it tried to do `UPDATE WHERE blah`, and matched 0 rows.

So, I learned, and now you can too.